machine_management/managing-infrastructure-machinesets.adoc

```diff
@@ -0,0 +1,66 @@
+[id="managing-infrastructure-machinesets"]
+= Managing infrastructure machine sets
+include::modules/common-attributes.adoc[]
+:context: managing-infrastructure-machinesets
+
+toc::[]
+
+Infrastructure machine sets provide a way to create and manage infrastructure nodes, which allow you to isolate infrastructure workloads. Managing your infrastructure workloads on infra nodes has several benefits.
+
+include::modules/infrastructure-components.adoc[leveloffset=+1]
+
+== Workload placement for infra nodes
+
+The two primary reasons for infra nodes include:
+
+* Preventing incurring billing costs against subscription counts.
+* Separating maintenance workloads and management workloads.
+
+After creating an infrastructure machine set, the new hosted infrastructure nodes have both the `worker` and `infra` roles applied to them. The `worker` role is required when the node is created, and is also used during node upgrades. Nodes with the `infra` role applied are not counted toward the total number of subscriptions that are required to run the environment, even when the `worker` role is also applied. The `worker` role can be removed, but it requires additional procedures to reconfigure the associated machine config pool to manage node upgrades.
```

Comment:
The core issue with this is that, infra component and 'customer workloads' can by default run on 'infra' nodes. 

Some customers may not want this! We need to elaborate more in this section (or provide a link to instructions or a set of materials) that outline how to isolate infra nodes to infra components, only.